packages/p2p/src/components/my-profile/block-user/__tests__/block-user.spec.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { act, screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useStores } from 'Stores/index';
+import { my_profile_tabs } from 'Constants/my-profile-tabs';
+import { localize } from 'Components/i18next';
+import BlockUser from '../block-user';
```

Comment:
can we move this import down

Reply:
do you mean the constants?

Reply:
```suggestion
import { localize } from 'Components/i18next';
import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
import { my_profile_tabs } from 'Constants/my-profile-tabs';
```

like this?

Reply:
yeah.. i just meant the order.. the relative imports should be at the bottom, right..